packages/logging-system/src/SetLocalStorage.ts

```diff
@@ -1,36 +1,32 @@
 import { LogRequestList, LogResponse, LogType } from './types/LogType';
 
 export const SetLocalStorageClear = () => {
-  const list: any[] = [];
+  const list: LogType[] = [];
   localStorage.setItem('yls-web', JSON.stringify(list));
```

Comment:
```suggestion
  localStorage.setItem('yls-web', "[]");
```
이거 그냥 이렇게 해도 되지 않나요?